LunrCoreLmdbTests/SearchTests.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Lunr;
+using LunrCoreLmdb;
+using Xunit;
+using Index = Lunr.Index;
+
+namespace LunrCoreLmdbTests
+{
+    public class SearchTests
+    {
+        [Fact]
+        public async void Can_populate_from_existing_index() => await WithDelegatedIndex(Assert.NotNull);
+
+        [Fact]
+        public async Task SearchWithBuildTimeFieldBoostsNoQueryBoost()
+        {
+            await WithDelegatedIndex(async idx =>
+            {
+                Result result = (await idx.Search("professor").ToList()).First();
+                // b ranks highest
+                Assert.Equal("b", result.DocumentReference);
+            });
+        }
+
+        private async Task WithDelegatedIndex(Action<DelegatedIndex> action)
+        {
+            var plain = await GetPlainIndex();
+
+            var path = $"{Guid.NewGuid()}";
```

Comment:
This is a little gratuitous :D

Reply:
@bleroy Can you give me more detail for this? I'm just not sure what you mean. There is certainly a lot of code duplication due to trying to avoid touching your original implementation, but I did want to run tests, and your pre-populated indexes are good for that. If you're referring to the interpolation, that's just a bad habit of mine.

Reply:
(I removed the silly interpolation)

Reply:
It was the silly interpolation :)